openduck-py/openduck_py/routers/voice.py

```diff
@@ -285,10 +285,17 @@ async def start_response(
             messages = chat.history_json["messages"]
             messages.append(new_message)
 
-            response = await acompletion(
-                CHAT_MODEL, messages, temperature=0.3, stream=True
-            )
-
+            # NOTE(zach): retries
+            response = None
+            for _ in range(3):
+                try:
+                    response = await acompletion(
+                        CHAT_MODEL, messages, temperature=0.3, stream=True
+                    )
+                except Exception:
+                    raise
```

Comment:
should this be `continue` instead of `raise`? `raise` will throw and exception without retrying right?

Reply:
or `pass`

Reply:
ooh good call. need to raise if all of them fail though. fixing now

Reply:
[bd1e794](https://github.com/uberduck-ai/openduck/pull/76/commits/bd1e794983f3389a500263ea2373aff358553210)